sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_counts_v1/query.sql

```diff
@@ -0,0 +1,300 @@
+WITH table_counts AS (
+  SELECT
+    'account_customers' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_account_customers_v1` FOR SYSTEM_TIME AS OF TIMESTAMP(
+      @as_of_date,
```

Comment:
I'd leave this decision to @clouserw.

Reply:
I'm having a hardtime wrapping my head around that comment.  I'm up for whatever you recommend and/or whatever has precedent.

Reply:
To keep things more straightforward I'd suggest going with the first option I proposed (midnight at the end of the partition date), and you'll just need to keep in mind that the counts are actually based on FxA data from _early_ on that date (~01:30 UTC) because that's when the FxA database syncs run.

Reply:
OK, I'm going to merge this suggestion and we can see how this works.

Reply:
> This also has the nice property that if we ever increase the FxA database sync frequency (e.g. to hourly) then this daily monitoring ETL will still work as intended.

I just realized this isn't exactly true, because if it's running hourly `@as_of_date + 1` would often be in the future.  So if this needs to run more frequently than daily at some point then we'd have to rethink the approach.